src/vmq_reg.erl

```diff
@@ -219,6 +219,18 @@ register_subscriber(SessionPid, SubscriberId,
             {ok, SessionPresent2, QPid}
     end.
 
+block_until_migrated(_, []) -> ok;
```

Comment:
We should put it to a different location inside vmq_reg.
Should we add a timeout/max-retry mechanism?